ci/build/build-standalone-release.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+#!/usr/bin/env bash
```

Comment:
Looks like an extra hashbang snuck in.
```suggestion
```

Reply:
Yup - already sending an amend completely removing any diff on that file as not needed anymore.

Reply:
Awesome~